md_mermaid.py

```diff
@@ -85,7 +85,7 @@ def run(self, lines):
 class MermaidExtension(Extension):
     """ Add source code hilighting to markdown codeblocks. """
 
-    def extendMarkdown(self, md, md_globals):
```

Comment:
md_globals=None for backwards compat, then there's no reason not to take this change.